app/src/main/java/org/wikipedia/analytics/eventplatform/AppearanceSettingInteractionEvent.kt

```diff
@@ -0,0 +1,40 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.wikipedia.Constants.InvokeSource
+import org.wikipedia.auth.AccountUtil
+import org.wikipedia.theme.Theme
+
+class AppearanceSettingInteractionEvent(private var source: InvokeSource) {
```

Comment:
All constructor parameters in all of the Event classes and all Impl classes can be `val` instead of `var`.